data/Dels/Dels missions.txt

```diff
@@ -925,6 +927,44 @@ mission "Dels Officer: Escort Convoy"
 	on visit
 		dialog phrase "generic arrived-without-npc dialog"
 
+mission "Dels Officer: Bad Officer 1"
+	repeat
+	landing
+	to offer
+		"dels officer: bad officer" >= 10
+	on offer
+		fine 10000
+		"dels officer: demerit count" += "dels officer: bad officer"
+		"dels officer: bad officer" = 0
+		dialog `Suddenly, you recieve a message from navy command! It reads, "Officer <first>, we are highly disappointed in you, as you have received many demerits. Because of this, you are being fined 10,000 credits."`
+
+mission "Dels Officer: Bad Officer 2"
+	landing
+	to offer
+		"dels officer: demerit count" > 100
+	on offer
+		fine 100000
+		dialog `Suddenly, you recieve a message from navy command! It reads, "Officer <first>, you have over one hundered demerits. This is a great disapointment. Thus, you will be fined 100,000 credits."`
+
+mission "Dels Officer: Bad Officer 3"
+	landing
+	to offer
+		"dels officer: demerit count" > 500
+	on offer
+		fine 1000000
+		dialog `Suddenly, you recieve a message from navy command! It reads, "Officer <first>, you must STOP your infuriating behavior if you want to stay in the Kaote navy! As one final repirmand, you will be fined ONE MILLION credits."`

```

Comment:
This dialog is very unprofessional and doesn't fit a proper government navy. Also 1 million isn't that and we don't need 4 stages, charge 1 millions on the 2nd offence and kick out on the third.

Reply:
Would one mission where you get kicked out be better?

Reply:
Not too sure what you meant by that.

Reply:
I meant something more like removing all missions except for a tweaked final mission (the one where you lose your license).